doc/NEWS

```diff
@@ -1,5 +1,7 @@
 Major changes from 1.9.0-jumbo-1 (May 2019) in this bleeding-edge version:
 
+- Added support for Cardano legacy's encrypted secret keys. [ilap; 2022]
+
```

Comment:
This should be at the end of the list, around line 256.

Reply:
Thx, on it.